taskcluster/script.py

```diff
@@ -212,11 +212,12 @@ def main():
                             stderr=subprocess.STDOUT)
     while True:
         line = proc.stdout.readline()
-        line_len = len(line)
+        decoded_line = line.decode()
```

Comment:
str.decode only exists on python2 not python3.

Reply:
readline returns a bytes object in py3. bytes.decode does exist.

I've tested it locally.

Reply:
```
➜  ~  python3
Python 3.7.5 (default, Nov  1 2019, 02:16:32) 
[Clang 11.0.0 (clang-1100.0.33.8)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> c = b'Test Byte Object'
>>> c.decode()
'Test Byte Object'
>>>
```

Reply:
```
➜  ~  python3
Python 3.7.5 (default, Nov  1 2019, 02:16:32) 
[Clang 11.0.0 (clang-1100.0.33.8)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import subprocess
>>> proc = subprocess.Popen(['ls'],stdout=subprocess.PIPE)
>>> line = proc.stdout.readline()
>>> line
b'Applications\n'
>>> 
```

Reply:
Thanks!